recipe/meta.yaml

```diff
@@ -11,18 +11,22 @@ source:
   sha256: 957f669d4922f92c171ba01bef7f29410668db09f6c02111e22b2bce446f3285
 
 build:
-  number: 0
-  skip: true  # [py2k]
+  number: 1
+  noarch: python   # [use_noarch]
+  track_features:
+    - fonttools_no_compile    # [use_noarch]
```

Comment:
@isuruf You mean the noarch version has the `treack_features`?  

Sorry if I am being difficult, I'm try to understand what is going on.